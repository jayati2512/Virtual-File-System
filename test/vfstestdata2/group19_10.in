create vfs_1 -453
create vfs_1 1
create vfs_1 1
mountvfs vfs_1
mountvfs vfs_1
makedir /home dir1
movedir /home /non_existent_directory 
makedir /home /dir2/dir3
deletedir /home/dir1/file1
addfile /home/dir1/ file1.txt /usr/include/stdio.h
addfile /home/dir2/ file1.txt /usr/include/stdio.h
deletedir 7675
deletedir /home/dir1
deletedir dir3
removefile /home/dir2/file2
unmountvfs vfs_1
unmountvfs vfs_1
