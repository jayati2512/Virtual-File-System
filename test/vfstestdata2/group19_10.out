CANNOT_CREATE_DATAFILE
createvfs_SUCCESS
DATA_FILE_ALREADY_EXISTS
mountvfs_SUCCESS
VFS_ALREADY_MOUNTED
makedir_SUCCESS
CANNOT_FIND_SPECIFIED_DESTINATIONDIR
makedir_SUCCESS
CANNOT_FIND_SPECIFIED_PATH_OR_FILE
addfile_SUCCESS
addfile_SUCCESS
INVALID_DIRNAME
deletedir_SUCCESS
INVALID_DIRNAME
removefile_SUCCESS
unmountvfs_SUCCESS
VFS_NOT_MOUNTED
